tasks/tangd-custom-port.yml

```diff
@@ -0,0 +1,53 @@
+---
+# This task tells SELinux for the port that we want the tangd service to use
+- name: Allow the custom port for tangd_port_t in SELinux
+  community.general.seport:
+    ports: "{{ nbde_server_port }}"
+    proto: tcp
+    setype: tangd_port_t
+    state: present
+  when: ansible_selinux.status == "enabled"
+
+# This block creates the override file for systemd with the new port that we have requested
+- name: Create override file
+  block:
+    # This task checks if the directory /etc/systemd/system/tangd.socket.d exist and registers the value in systemd__system__tangd_socket
+    - name: Check if directory /etc/systemd/system/tangd.socket.d exist
+      stat:
+        path: /etc/systemd/system/tangd.socket.d
+      register: systemd_system_tangd_socket
+
+    # This tasks Create the /etc/systemd/system/tangd.socket.d directory if it does not exist
+    - name: Create a directory if it does not exist
+      file:
+        path: /etc/systemd/system/tangd.socket.d
+        state: directory
+        mode: '0755'
+      when: systemd_system_tangd_socket.stat.exists
+
+    # This tasks creates the file with the port entry that we want tangd to listen to 
+    - name: Creates the file with the port entry that we want tangd to listen to 
+      blockinfile:
+        path: /etc/systemd/system/tangd.socket.d/override.conf
+        block: |
+          [Socket]
+          ListenStream={{ nbde_server_port }}
```

Comment:
`ListenStream` is a list, so if you just specify a port here, it will be like you are adding this new port to the list. You should first clear this list with an empty `ListenStream=` line prior to setting the new port. Check out https://www.freedesktop.org/software/systemd/man/systemd.socket.html#Options for more info.

Also, it would be great if you could add a test to make sure the server is using the new port (and ideally make also sure we are not using in the old one)